backend/src/main/java/com/eyetrade/backend/model/dto/ArticleDto.java

```diff
@@ -0,0 +1,16 @@
+package com.eyetrade.backend.model.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class ArticleDto {
+    public String authorEmail;
```

Comment:
We should discuss about whether author's email, name, surname should be in dto (However, it must be in resource). Because ArticleDto comes from the user who creates the article and she does know only the token unless she make a new request to gather that info. But it is not a big problem for now.